metrics/bigquery.py

```diff
@@ -124,24 +216,10 @@ def main(configs, project, bucket_path):
                 config = yaml.safe_load(config_raw)
             if not config:
                 raise ValueError('invalid yaml: %s.' % path)
-            metric = config['metric'].strip()
-            validate_metric_name(metric)
-            run_metric(
-                project,
-                bucket_path,
-                metric,
-                config['query'],
-                config['jqfilter'],
-                config.get('jqmeasurements'),
-                influx,
-            )
-        except (
-                ValueError,
-                KeyError,
-                IOError,
-                subprocess.CalledProcessError,
-                influxdb.Error,
-            ):
+            config['metric'] = config['metric'].strip()
+            validate_metric_name(config['metric'])
+            queryer.run_metric(config)
+        except Exception: # pylint: disable=broad-except
```

Comment:
Please catch the specific exceptions you are expecting

Reply:
done